lib/cryptoexchange/client.rb

```diff
@@ -8,6 +8,8 @@ def initialize(ticker_ttl: 3)
     def pairs(exchange)
       pairs_classname = "Cryptoexchange::Exchanges::#{StringHelper.camelize(exchange)}::Services::Pairs"
       Object.const_get(pairs_classname).new.fetch
+    rescue HTTP::ConnectionError, HTTP::TimeoutError, JSON::ParserError
```

Comment:
@Swingcloud actually, where is `begin`?

Reply:
@dominicwong617  method "def" can serve as a "begin" statement
or we can put begin on it I guess, but it works the same 

https://stackoverflow.com/questions/1542672/how-does-one-use-rescue-in-ruby-without-the-begin-and-end-block

Reply:
@Swingcloud TIL, no worries